applications/app/views/fragments/crosswords/crosswordMetaHeader.scala.html

```diff
@@ -42,11 +42,8 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(crosswordPage.c
             </div>
 
             @crosswordPage.crossword.instructions.map { instructions =>
-                <div class="crossword__instructions">
-                    <strong>Special instructions:</strong> @Html(instructions)
-                </div>
+                <div class="crossword__instructions"><strong>Special instructions:</strong> @Html(instructions)</div>
```

Comment:
Having put `white-space: pre-line` on the div here, the newlines between the tags becomes important and without changes, extra whitespace above and below the instructions would appear